test/src/test/java/hudson/model/RunTest.java

```diff
@@ -219,7 +219,7 @@ public FullNameChangingProject newInstance(ItemGroup parent, String name) {
         }
     }
     
-    private static final class Mgr extends ArtifactManager {
+    public static final class Mgr extends ArtifactManager {
```

Comment:
If we didn't make this `public`, then the `public` keyword for the `Factory` constructor on line 233 would be redundant, forcing us to remove it to satisfy Checkstyle's `RedundantImport` check, which would then break a different check which demands that anything annotated with `@DataBoundConstructor` be `public`. Looking at how other test classes deal with this problem, I see that most test code that uses `@DataBoundConstructor` is nested within a `public` parent class, so I opted to follow the same strategy here.